app/models/webpush_subscription.rb

```diff
@@ -0,0 +1,24 @@
+class WebpushSubscription < ApplicationRecord
+  belongs_to :user
+  belongs_to :account
+  validates :endpoint, presence: true
+  validates :p256dh_key, presence: true
+  validates :auth_key, presence: true, uniqueness: true
+
+  def send_notification(message)
+    begin
+      Webpush.payload_send(
+        message: JSON.generate(message),
+        endpoint: endpoint,
+        p256dh: p256dh_key,
+        auth: auth_key,
+        vapid: {
+          private_key: ENV['VAPID_PRIVATE_KEY'],
+          public_key: ENV['VAPID_PUBLIC_KEY']
+        }
+      )
+    rescue Webpush::ExpiredSubscription => e
+      return {error: e.message}
```

Comment:
Aqui vale a pena logarmos o erro algo assim:
```suggestion
    rescue Webpush::ExpiredSubscription => e
      Rails.logger.error("#{self.class} - #{e.message} - #{e.backtrace.join("\n")} - Account: #{account}")
      return {error: e.message}
```